sport/app/football/views/wallchart/groupTableEmbed.scala.html

```diff
@@ -0,0 +1,75 @@
+        <div class="football-group__container football-group__container-embed table--hide-from-importance-3">
+            @round.name.map{ name =>
+                <div class="football-group__table">
+                    @football.views.html.tablesList.tableView(competition, model.Group(round, leagueTableEntries),
+                        heading = round.name,
+                        headingLink = groupTag(competition.id, round),
+                        striped = true,
+                        withCrests = true
+                    )
+                </div>
+                <div class="football-group__matches">
+                @{matches.matchesGroupedByDateAndCompetition.zipWithRowInfo.map { case ((date, competitionMatches), info) =>
+                    competitionMatches.map { case (competition, matches) =>
+                        football.views.html.matchList.matchesList(matches, competition, date,
+                            linkToCompetition = false,
+                            heading = if(info.isFirst) Option(("Fixtures and results", None)) else None
+                        )
+                    }
+                }}
+                </div>
+            }
+        </div>
+    </li>
+}
+}
+
+
+
+<!DOCTYPE html>
+
+<html class="js-off is-not-modern id--signed-out" lang="en-GB">
+    <head>
+        <meta name="robots" content="noindex, nofollow" />
+        @fragments.head(Some("football"), Html(""))(page, request, context)
+        <script>guardian.config.page.isFootballWorldCup2014 = true;</script>
+        </head>
+    <body id="top" itemscope itemtype="http://schema.org/WebPage">
+        <div id="js-context" class="football-wallchart-embed">
+            <div class="l-side-margins wc-overview">
+                    <div class="facia-container facia-container--layout-content">
+                        <div class="container">
+                            <div class="facia-container__inner">
+                                <div class="container__border hide-on-mobile"></div>
+                                <div class="fc-container__header">
+                                    <h3 class="container__title">
+                                        <span class="container__title__label u-text-hyphenate">@groupTable._1.name</span>
+                                    </h3>
+                                </div>
+                                <div class="container__body">
+                                    <div data-link-name="@competition.fullName groups">
+                                        <ul class="football-groups u-unstyled u-cf">
+                                            @footballGroup(groupTable._1, groupTable._2)
+                                        </ul>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                @fragments.analytics.base()(page, request, context)
```

Comment:
Will the analytics included here be able to understand this isn't a normal pageview? Should we strip them out?

Reply:
See [above comment](https://github.com/guardian/frontend/pull/23822#discussion_r636223318) about ripping out this in a future PR